lego/apps/events/views.py

```diff
@@ -436,6 +437,7 @@ def create(self, request, *args, **kwargs):
             registration.feedback = feedback
             registration.save(current_user=current_user)
             transaction.on_commit(lambda: async_register.delay(registration.id))
+        check_event_related_single_user(current_user)
```

Comment:
I don't this this would work if the user is admin registered. A more robust way to do this would be to add a hook/signal to whenever a `Registration` is saved.
https://medium.com/django-musings/django-s-hooks-and-signals-6166369d798b
https://docs.djangoproject.com/en/5.1/topics/signals/
https://docs.djangoproject.com/en/5.1/ref/signals/

I also think this should run async in a celery task, as some of the checks seem like they might be expensive

Reply:
This is fixed now.